### ISSUE TITLE
Critical: Opening an application by opening it's document stored on floppy, will result in system hang

### ISSUE THREAD

**Issue by MauriceMEGA65** [bluewaysw/geos#21]
As example, I open GEOWRITE and at startup when it asks me to create a document, I select the floppy drive. 
Document is created (testdoc, see screenshot) and I close GEOWRITE.

![mega65-screen-000066](https://github.com/bluewaysw/geos/assets/75681098/149f3e54-bd96-4552-8258-bfd486fadf6c)

Now, when double click on TESTDOC, it seems GEOS is trying to open GEOWRITE. Floppy drive is reading nicely as it sounds. But after some time when I expect GEOWRITE to open, the screen turns all grey and GEOS seems to hang. Reset needed to get out of the situation.

**Comment by bluewaysw** [comment]
Is the hanging grey screen reproducable? For me GeoWrite comes up in the test case but shows Error I:21/swapping geoWrite.... whatever this means.

**Comment by MauriceMEGA65** [comment]
Sorry, cannot reproduce..
For some reason the floppy is not being read anymore, I get "function aborted due to disk error $0D".
Same when trying to format the floppy.

I just updated to GEOS beta r4, and that worsened the situation for me:
-same disk error
-mouse less stable / jumpy (update: after a few power cycles mouse is more stable...)
-format the floppy crashes GEOS and SuperDebugger screen is presented.

**Comment by bluewaysw** [comment]
I not changed anything related to fromat, that is issues #25. 
r4 shoud not change anything related to the mouse.
But I can confirm the problem with GEOWRITE.... this is work in progress. GEOPAINT on the other sie does it well for me.

**Comment by MauriceMEGA65** [comment]
Found the issue why the internal floppy is no longer working in GEOS65: the option DISABLE DISK IMAGE DRIVE NOISE in MEGA65 Configuration, is messing up the ability to access the internal floppy drive in GEOS65. When enabled (so getting disk noise), and all works fine again in GEOS65.

**Comment by bluewaysw** [comment]
@MauriceMEGA65 can you confirm the issues has been fixed?

**Comment by MauriceMEGA65** [comment]
Would love to. Do you have a D81 for me that contains the updated GEOS plus has the apps one it?

**Comment by bluewaysw** [comment]
@MauriceMEGA65 do you have you MEGA65 connected to the Internet via ethernet connection?

**Comment by MauriceMEGA65** [comment]
Yes, I have an ethernet connection. Can I pull it that way from your host or so?
